src/media/js/helpers_local.js

```diff
@@ -75,6 +75,17 @@ define('helpers_local',
         return array.slice(begin, end);
     };
 
+    filters.fileSize = function(int) {
+        var bytes = parseInt(int, 10);
+        if (bytes < 1024^2) {
+          return +(bytes/1024).toFixed(2) + ' KB';
+        } else if (bytes < 1024^3) {
+          return +(bytes/1024^2).toFixed(2) + ' MB';
+        } else {
+          return +(bytes/1024^3).toFixed(2) + ' GB';
+        }
+    };
```

Comment:
This is syntactically a little confusing. This will help explain:

    >>> (1.1010).toFixed(4);
    "1.1010"

    >>> +(1.1010).toFixed(4);
    1.101

    >>> +(1.1010).toFixed(3);
    1.101

    >>> +(1.1010).toFixed(2);
    1.1

    >>> +(1.1010).toFixed(1);
    1.1

    >>> +(1.1010).toFixed(0);
    1